plugins/woocommerce/package.json

```diff
@@ -107,6 +107,7 @@
 					"command": "test:php:env",
 					"changes": [
 						"client/admin/config/*.json",
+						"composer.json",
```

Comment:
Unexpectedly all PHP/E2E jobs are triggered, while I expected just one. @adimoldovan, what do I miss?

Reply:
I wonder if composer.json should also be included by default 🤔

Reply:
Gotcha! Good question; my thinking was to run a single php unit test job to validate composer config changes, but we can consider updating other entries here to treat changes in both composer.lock and composer.json files equally.

Reply:
Replicated the change in other entries: 8f806ae36e8e98d3da850eda334d716c5f9db5f2